src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    width: $stars-width;
+    height: $stars-height;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &__star:last-child {
```

Comment:
why did you insert `&__star:last-child { ` inside  `&__star {` ?

they should be on the same level :)